openshift4/templates/minespace.dc.yaml

```diff
@@ -81,6 +81,10 @@ parameters:
     displayName: AppName
     description: Application name for grouping pods in Openshift
     value: Stand-Alone
+  - name: VCAUTHN_PRES_REQ_CONF_ID
+    displayName: VC-AUTHN-PRES-REQ_CONF_ID
+    description: Presentation Requestion Configuration Id for KC to foward to vc-authn Identity Provider
+    value: minespace-access-0.1-dev
```

Comment:
@jon-funk How/Where do we define the parameters into the DC's? this should be different for each environment.